examples/kubernetes-ingress/scripts/helpers.bash

```diff
@@ -3,20 +3,36 @@
 if [[ -n "${IPV6_EXT}" ]]; then
```

Comment:
Add logs using tests/helpers.bash:log so users can follow what's going on in case they hit an issue while the script runs. Given that this file wasn't heavily modified during this PR, you can file this as an enhancement ticket and reference a GH issue in a comment in this file.